File: Programming Basics with Python/03 Conditional statements advanced/More exercises/01_Match_Tickets.py
budget = float(input())
category = input()
people = int(input())
transport = 0
tickets = 0
if category == "VIP":
    tickets = 499.99 * people
elif category == "Normal":
    tickets = 249.99 * people
if 1 <= people <= 4:
    transport = budget * 0.75
elif 5 <= people <= 9:
    transport = budget * 0.60
elif 10 <= people <= 24:
    transport = budget * 0.50
elif 25 <= people <= 49:
    transport = budget * 0.40
elif people >= 50:
    transport = budget * 0.25
money_for_tickets = budget - transport
result = abs(money_for_tickets - tickets)
if money_for_tickets >= tickets:
    print(f"Yes! You have {result:.2f} leva left.")
else:
    print(f"Not enough money! You need {result:.2f} leva.")

File: JS Basics/03 Nested Conditional Statements - More Exercises/01. Match Tickets.js
function match(budget, category, fans) {
    budget = Number(budget);
    fans = Number(fans);
    let transport = 0;
    let tickets = 0;
    if (fans <= 4) {
        transport = budget * 0.75;
    }
    else if (5 <= fans && fans <= 9) {
        transport = budget * 0.6;
    }
    else if (10 <= fans && fans <= 24) {
        transport = budget * 0.5;
    }
    else if (25 <= fans && fans <= 49) {
        transport = budget * 0.4;
    }
    else if (fans >= 50) {
        transport = budget * 0.25;
    }
    if (category == "VIP") {
        tickets = fans * 499.99;
    }
    else {
        tickets = fans * 249.99;
    }
    let costs = tickets + transport;
    let difference = Math.abs(costs - budget);
    if (budget >= costs) {
        console.log(`Yes! You have ${difference.toFixed(2)} leva left.`)
    }
    else {
        console.log(`Not enough money! You need ${difference.toFixed(2)} leva.`)
    }
}
match(30000, "VIP", 49)